### PR TITLE
Added onclickListner for recyclerview 

### PR DESCRIPTION
**Pull request by Rohit1173**
**When we click on an event on the recycler view a new fragment appears which contains the data of the respective event**
**Attaching a video for better understanding**

https://user-images.githubusercontent.com/78897906/146687313-57becbc8-3276-4851-b9ee-bc66966b542a.mp4

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (yash10019coder) [inline .idea/misc.xml]: revert this file changes

**Review by yash10019coder — CHANGES_REQUESTED** [review]
LGTM @Rohit1173 just revert the misc.xml changes.

File: .idea/misc.xml (hunk 1 above)
Comment (Rohit1173) [reply .idea/misc.xml]: @yash10019coder  the misc.xml have been added by default so I have deleted the file after committing. Is it necessary to add that back?

Comment (yash10019coder) [reply .idea/misc.xml]: revert back

Comment (Rohit1173) [reply .idea/misc.xml]: done

**Pull request merged**